### PR TITLE
Add documentation for Frank Energie

### PR DESCRIPTION
**Pull request by DCSBL**
## Proposed change

Add documentation for the new Frank Energie integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/103422
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4856
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/frank_energie.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ha_codeowners:
+  - '@bajansen'
+  - '@DCSBL'
+ha_domain: frank_energie
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The Frank Energie integration connects the [Frank Energie](https://www.energyzero.nl/) API platform with Home Assistant.
```
Comment (klaasnicolaas) [inline source/_integrations/frank_energie.markdown]: Is it using the energyzero api?

Comment (DCSBL) [reply source/_integrations/frank_energie.markdown]: Whoops 😅

Comment (DCSBL) [inline source/_integrations/frank_energie.markdown]:
```suggestion
The Frank Energie integration connects the [Frank Energie](https://www.frankenergie.nl/) API platform with Home Assistant.
```

Comment (DCSBL) [reply source/_integrations/frank_energie.markdown]: Fixed, thanks 👍

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**